README.md

```diff
@@ -716,6 +716,9 @@ Comments that contradict the code are much worse than no comments.
 Always make a priority of keeping the comments up-to-date with code changes!
 
 Comments should be complete sentences.
+Long sentences should be wrapped at 92 characters and not earlier.
```

Comment:
What if the 92'd character is midword?
What if its mid-quote?
what if there is a comma at character 80?